db/db_impl_secondary.cc

```diff
@@ -204,11 +204,15 @@ Status DBImplSecondary::RecoverLogFiles(
             record.size(), Status::Corruption("log record too small"));
         continue;
       }
+      SequenceNumber seq = versions_->LastSequence();
       WriteBatchInternal::SetContents(&batch, record);
+      SequenceNumber seq_of_batch = WriteBatchInternal::Sequence(&batch);
+      if (seq_of_batch < seq) {
```

Comment:
would be nice to add some comments to explain why we are doing this check

Reply:
Sure.